### ISSUE TITLE
Add changed control into the Climatic > Prfepare > Summary dialog

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#9135]
@N-thony and @lilyclements I hope this is a simple change now?

Here is the climatic summary dialog.

![image](https://github.com/user-attachments/assets/cfa3a2ea-c62c-4369-84cc-b7f3391a18ee)

I trust it is using the same code, for most bits, as the Prepare > Data Reshape > Summary dialog?

So please could you include the new enabled Drop Uniused Levels control?  It should also be ticked by default here?  Please report if it seems complicated?  That may mean we need to think harder first.

**Comment by lilyclements** [comment]
@rdstern yes - just checked and this is a simple change. Should be as simple as enabling the checkbox, and setting `drop = FALSE` when unchecked.

**Comment by derekagorhom** [comment]
@rdstern , @N-thony I am closing this Issue
